src/main/java/org/bahmni/reports/extensions/util/Templates.java

```diff
@@ -0,0 +1,189 @@
+        columnStyle = stl.style(rootStyle).setVerticalAlignment(VerticalAlignment.MIDDLE).setHorizontalAlignment(HorizontalAlignment
+                .CENTER);
+        columnTitleStyle = stl.style(columnStyle)
+                .setBorder(stl.pen1Point())
+                .setHorizontalAlignment(HorizontalAlignment.CENTER)
+                .setBackgroundColor(Color.LIGHT_GRAY)
+                .bold();
+        groupStyle = stl.style(boldStyle)
+                .setHorizontalAlignment(HorizontalAlignment.LEFT);
+        subtotalStyle = stl.style(boldStyle)
+                .setTopBorder(stl.pen1Point());
+
+        minimalColumnStyle = stl.style().setRightBorder(stl.pen1Point());
+
+        StyleBuilder crosstabGroupStyle = stl.style(columnTitleStyle);
+        StyleBuilder crosstabGroupTotalStyle = stl.style(columnTitleStyle)
+                .setBackgroundColor(new Color(170, 170, 170));
+        StyleBuilder crosstabGrandTotalStyle = stl.style(columnTitleStyle)
+                .setBackgroundColor(new Color(140, 140, 140));
+        StyleBuilder crosstabCellStyle = stl.style(columnStyle)
+                .setBorder(stl.pen1Point());
+
+        TableOfContentsCustomizerBuilder tableOfContentsCustomizer = tableOfContentsCustomizer()
+                .setHeadingStyle(0, stl.style(rootStyle).bold());
+
+        reportTemplate = template()
+                .setLocale(Locale.ENGLISH)
+                .setColumnStyle(columnStyle)
+                .setColumnTitleStyle(columnTitleStyle)
+                .setGroupStyle(groupStyle)
+                .setGroupTitleStyle(groupStyle)
+                .setSubtotalStyle(subtotalStyle)
+                .highlightDetailEvenRows()
+                .crosstabHighlightEvenRows()
+                .setCrosstabGroupStyle(crosstabGroupStyle)
+                .setCrosstabGroupTotalStyle(crosstabGroupTotalStyle)
+                .setCrosstabGrandTotalStyle(crosstabGrandTotalStyle)
+                .setCrosstabCellStyle(crosstabCellStyle)
+                .setTableOfContentsCustomizer(tableOfContentsCustomizer);
+
+        excelReportTemplate = template()
+                .setLocale(Locale.ENGLISH)
+                .setColumnStyle(columnStyle)
+                .setColumnTitleStyle(columnTitleStyle)
+                .setGroupStyle(groupStyle)
+                .setGroupTitleStyle(groupStyle)
+                .setSubtotalStyle(subtotalStyle)
+                .highlightDetailEvenRows()
+                .crosstabHighlightEvenRows()
+                .setCrosstabGroupStyle(crosstabGroupStyle)
+                .setCrosstabGroupTotalStyle(crosstabGroupTotalStyle)
+                .setCrosstabGrandTotalStyle(crosstabGrandTotalStyle)
+                .setCrosstabCellStyle(crosstabCellStyle)
+                .ignorePagination()
+                .setTableOfContentsCustomizer(tableOfContentsCustomizer);
+
+        currencyType = new CurrencyType();
+
+        HyperLinkBuilder link = hyperLink("http://www.dynamicreports.org");
```

Comment:
What is this URL for?

Reply:
This URL was part of the existing code in the Templates class in bahmni reports which contains the reports ui styles and formats.  I have removed the class from this repository and created the styles manually in the icd10ResultSetExtension class where they are needed. 

Below is the code snippet for the ui styles created.
```
private static class reportStyle {
        {
            columnStyle = stl.style().setPadding(2)
                    .setVerticalAlignment(VerticalAlignment.MIDDLE)
                    .setHorizontalAlignment(HorizontalAlignment.CENTER);
        }
    }
```